### DISCUSSION TITLE
steam not opening

### DISCUSSION THREAD

**Discussion by vyrx-dev** [prasanthrangan/hyprdots#1377] [Q&A] [answered]
![240424_00h06m19s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/139481531/dd23fe01-a1f6-4d46-845c-a91ebc2a5429)

**Comment by simplemachine-zz** [comment, marked as answer]
@theamit-969 https://github.com/ValveSoftware/steam-for-linux/issues/10980#issuecomment-2187324303
